docs/guides/installing-cedar/README.md

```diff
@@ -351,4 +351,130 @@ export default {
 </style>
 ```
 
+## Developing for Mobile
+
+Currently development for iOS and Android is through the usage of Cedar Tokens as documented below.  
+
+Our mobile development packages include fonts that are licensed and proprietary to REI. For more information or questions regarding Cedar’s support for external consumers using the mobile development packages on GitHub or NPM, please reach out to [cedar@rei.com](mailto:cedar@rei.com).
```

Comment:
This line also seems to apply to the main cedar library? Or no because the fonts are fetched from the REI CDN?

Reply:
it is specific to Mobile as the fonts need to be physically packaged up with their tokens for them to be able to use them - and web consumers will include the fonts in their projects separately via cdr-fonts.css  --- that said this really may not be very useful and our mobile team may want to author this part themselves as some of the process is changing